packages/blockchain-wallet-v4-frontend/src/scenes/Login/AccountUpgrade/3_3_1_GoogleAuthSetup/GoogleAuthSetup.tsx

```diff
@@ -22,10 +65,34 @@ const GoogleAuthSetup = (props) => {
           lineHeight='1.5'
         >
           <FormattedMessage
-            id='scenes.login.upgrade.upsell.header'
-            defaultMessage='GOOGLE_AUTH_SETUP'
+            id='scenes.login.upgrade.googleAuth.header'
+            defaultMessage='Set Up 2FA'
           />
         </CenteredTitle>
+        <CenteredTitle size='14px' weight={500} lineHeight='20px' color='grey900'>
+          <FormattedMessage
+            id='scenes.login.upgrade.googleAuth.text'
+            defaultMessage='With your Google Authenticator app, scan the QR code below to make a secure connection.'
+          />
+          <QrWrapper>Placeholder</QrWrapper>
+          <ClipboardWrapper>
+            {/* <CopyClipboard address={PLACEHOLDER} active={uiState.notificationActive} /> */}
+          </ClipboardWrapper>
+          <BadgesWrapper>
+            <Image name='badge-ios-appstore' width='160px' />
+            <Image name='badge-android-playstore' width='160px' />
+          </BadgesWrapper>
+        </CenteredTitle>
+
+        <ButtonNext
+          nature='primary'
+          data-e2e='nextButton'
+          fullwidth
+          height='48px'
+          onClick={() => {}}
+        >
+          <FormattedMessage id='scenes.login.upgrade.googleAuth.button' defaultMessage='Next' />
```

Comment:
There already exists an entry for "Next" message dont need to add a new one. `copy.next`

Reply:
but what if product needs to change the copy for this specific button in the future?

Reply:
Then you will need to create a new id. But this text already exists. If that change is required no matter the id you put you will need to make the change inside the code for modifying this text

Reply:
yes, we dont want to be duplicating translations unnecessarily for two reasons:
1) it bloats the app bundle size since strings cant be minified
2) it costs us money to retranslate the same text over and over

Reply:
Done, thanks juli!